README.md

```diff
@@ -61,13 +64,15 @@ _Note: may need to make **auto** folder_
 
 ##### Install build tools
 
-Debian/Ubuntu/Mint:   `$ sudo apt-get install build-essential libXmu-dev libXi-dev libgl-dev cmake`
+Debian/Ubuntu/Mint:   `$ sudo apt-get install build-essential libxmu-dev libxi-dev libgl-dev cmake`
```

Comment:
Ubuntu 17.10 Artful?

Reply:
Yes, but as I said it was only a misspelling of the packets name